pkg/controlplane/controller/leaderelection/leaderelection_controller_test.go

```diff
@@ -697,6 +698,7 @@ func TestController(t *testing.T) {
 			if err != nil {
 				t.Fatal(err)
 			}
+			controller.clock = fakeClock
```

Comment:
for this test this is counter-productive. We should not mix a real running controller (which we have here) with a fake clock.

Reply:
done